nixos/modules/services/editors/emacs.nix

```diff
@@ -83,9 +84,9 @@ in
       wantedBy = if cfg.startWithGraphical then [ "graphical-session.target" ] else [ "default.target" ];
     };
 
-    environment.systemPackages = [ cfg.package editorScript ];
+    environment.systemPackages = [ cfg.package ];
 
-    environment.variables.EDITOR = mkIf cfg.defaultEditor (mkOverride 900 "emacseditor");
+    environment.variables.EDITOR = mkIf cfg.defaultEditor (mkOverride 900 "emacsclient -ca '${cfg.alternativeEditor}'");
```

Comment:
Here are some issues I notice:
- Always passing `-c` prevents users from re-using existing frame(s).
- An empty string of `cfg.alternativeEditor` starts **another** Emacs daemon which is not managed by systemd.  I think it is a bad idea.  Users of this module probably want their daemon to be managed by system.
- The single quote in `-a '...'` causes issues if `cfg.alternativeEditor` contains environment variables.

Reply:
1. Imma be honest, I don't have enough experience with Emacs, but why reusing existing frames?
2. It only starts the daemon if it's not running (for example, when a standalone window manager user doesn't start `graphical-session.target` manually), otherwise nothing happens. 
3. Will it? Cuz the entire string will be in double quotes, won't it interpolate instantly?

Reply:
> why reusing existing frames?

For example, it is easier to move between buffers, windows and tabs in one frame.

> It only starts the daemon if it's not running

In that case, the old behaviour, i.e., launching a non-daemon Emacs, make more sense.  If you want a daemon, you should run `systemctl start emacs`.  Just my two cents.

> Will it?

My experiment says yes.

Reply:
2. Fair enough, I'll set the default altEditor value back to "emacs"